src/main/scala/io/iohk/ethereum/domain/Blockchain.scala

```diff
@@ -94,6 +103,8 @@ trait Blockchain {
 
   def save(hash: ByteString, evmCode: ByteString): Unit
 
+  def save(node: MptNode): Unit
```

Comment:
Ok...i'm not still sure but I don't think it's a blocker for merging. We can always refactor this.